src/assets/config/SimulationKernel.json

```diff
@@ -9,6 +9,7 @@
       "value": 1
```

Comment:
Would the strict integer format also apply for `localNumThreads` or is it not necessary there?

Reply:
Ok :+1:
However, I could not add it as a suggestion in a comment, because the line is too far away from your changes (and therefore deactivated/greyed out):
![grafik](https://user-images.githubusercontent.com/53972736/176720407-db939ca2-208e-4170-803b-01711da7851c.png)
Therefore, I needed to open another PR in your NEST Desktop fork.